### DISCUSSION TITLE
Synchronized browsing

### DISCUSSION THREAD

**Discussion by StoyanDimitrov** [doublecmd/doublecmd#1895] [Suggestions]
I would like to propose a new feature called "Synchronized browsing." Essentially, it is changing folders with the same name simultaneously in both panels of the application. That will ease navigation in synchronized folders and alike.

**Comment by Skif-off** [comment]
[cm_SyncChangeDir](https://doublecmd.github.io/doc/en/cmds.html#cm_SyncChangeDir), add button or hotkey.
